guides/source/asset_pipeline.md

```diff
@@ -680,18 +716,18 @@ A common pattern for using a CDN is to set your production application as the
 there is a cache miss, it will grab the file from your server on the fly and
```

Comment:
I think maybe the language is too informal/unclear here:

```suggestion
there is a cache miss, it will instead source the file from your server and
```